docs/current/README.md

```diff
@@ -53,49 +53,48 @@ Vaquero can run in multiple modes: `server`, `agent`, and `standalone`. This con
 **sample-standalone-config.yaml**
 ************************************************************
 ```
----
 ServerApi:
- Address: "127.0.0.1"
- Port: 24601
+  Address: "127.0.0.1"
+  Port: 24601
 AgentApi:
- InsecureAddr: "127.0.0.1"
- InsecurePort: 24602
+  InsecureAddr: "127.0.0.1"
+  InsecurePort: 24602
 AssetServer:
- Addr: "127.0.0.1"
- Port: 8080
- BaseDir: "/tmp/vaquero/files"
- Scheme: http
-DHCPMode: server
-DHCPCIDR: "127.0.0.1/16"
+  Addr: "127.0.0.1"
+  Port: 8080
+  BaseDir: "/tmp/vaquero/files"
+  Scheme: http
+DHCP:
+  Mode: server
+  Addr: "0.0.0.0"
 SavePath: "/tmp/vaquero"
 Gitter:
- Endpoint: "/postreceive"
- Timeout: 2
- Addr: "127.0.0.1"
- Port: 9090
+  Endpoint: "/postreceive"
+  Timeout: 2
+  Addr: "127.0.0.1"
+  Port: 9090
 GitHook:
- - ID: "vaquero-local"
-   Token: <GIT_TOKEN>
-   URL: "https://github.com/CiscoCloud/vaquero-examples"
-   Secret: <GIT_SECRET>
+  - ID: "vaquero-local"
+    Token: <GIT_TOKEN>
+    URL: "https://github.com/CiscoCloud/vaquero-examples"
+    Secret: <GIT_SECRET>
 SoT:
 - Git:
-     HookID: "vaquero-local"
-     ID: "vaquero-test"
-     Branch: local
+    HookID: "vaquero-local"
+    ID: "vaquero-test"
+    Branch: local
 Log:
- Level: debug
- Location: stdout
- LogType: text
+  Level: info
+  Location: stdout
+  LogType: text
 ```
 ************************************************************
 
 #### Explanation of config fields:
 - ServerApi: The user api for the server. Currently not implemented.
 - AgentApi: The vaquero-agent http server used to listen for vaquero server commands.
 - AssetServer: The asset server for vaquero agent used by each booting machine to get unattended scripts and kernels.
-- DHCPMode: One of two modes, `proxy` or `server`. Leaving the DHCP fields empty will disable all DHCP vaquero functionality.
-- DHCPCIDR: The CIDR managed by DHCP.
+- DHCP: One of two modes, `proxy` or `server`. DHCP endpoint address. Leaving the DHCP fields empty will disable all DHCP vaquero functionality.
```

Comment:
DHCPMode: `proxy`, `server` or `none` (maybe off?).  Leaving the DHCPMode field empty will disable all DHCP vaquero functionality.  

DHCPMode: Proxy enables ProxyDHCP. ProxyDHCP works with an existing DHCP Server to provide PXEBoot functionality, while leaving the managing and assigning of IP addresses to the other DHCP Server. ONLY enable this if you already have a DHCP server with entries for all the hosts in your Data Model.

DHCPMode: server runs vaquero as a DHCP server.  Vaquero does not manage free address pools or leases; it simply assigns based of the static configuration defined in the data model.